### PR TITLE
Account order - drag and drop accounts in account list is not smooth

### PR DESCRIPTION
**Pull request by saledjenic**
Corresponding `status-go` PR:
- https://github.com/status-im/status-go/pull/3763

This PR fixes:
#11508 
#11512 
#11513 

Functionallity:

https://github.com/status-im/status-desktop/assets/86303051/186b1168-30f8-4894-a65a-6877ff2b15fa

### CODE REVIEW

File: ui/app/AppLayouts/Profile/views/wallet/AccountOrderView.qml [hunk 1]
```diff
@@ -16,81 +16,95 @@ import utils 1.0
...
+        readonly property string walletAccountDnDKey: "status-wallet-account-item"
+        property int indexMoveFrom: -1
+        property int indexMoveTo: -1
     }
 
-    displaced: Transition {
-        NumberAnimation { properties: "x,y"; easing.type: Easing.OutQuad }
+    StatusBaseText {
+        Layout.fillWidth: true
+        text: accountsList.count > 1? qsTr("Move your most freqently used accounts to the top of your wallet list") :
```
Comment (caybro) [inline ui/app/AppLayouts/Profile/views/wallet/AccountOrderView.qml:38]: "frequently"

**Review by caybro — COMMENTED** [review]
Very nice!

**Review by caybro — APPROVED** [review]
I meant _really_ nice :)

File: ui/app/AppLayouts/Profile/views/wallet/AccountOrderView.qml (hunk 1 above)
Comment (saledjenic) [reply ui/app/AppLayouts/Profile/views/wallet/AccountOrderView.qml]: Updated, thanks! :)

**Review by alaibe — APPROVED** [review]

**Pull request merged**